charts/stable/qbittorrent/values.yaml

```diff
@@ -60,6 +61,7 @@ workload:
             QBT_PASSWORD: "{{ .Values.qbitportforward.QBT_PASSWORD }}"
             QBT_ADDR: '{{ printf "http://%v:%v" (include "tc.v1.common.lib.chart.names.fullname" $) .Values.service.main.ports.main.port }}'
             GTN_ADDR: '{{ printf "http://%v-gluetun:8000" (include "tc.v1.common.lib.chart.names.fullname" $) }}'
+          cmd: ./main.sh
```

Comment:
command

Reply:
Also I'd say use the full path instead of relative

Reply:
Not sure what you mean.

Reply:
Got it made the change.